FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Models/DicomApiOption/BaseDicomApiOptions.cs

```diff
@@ -3,21 +3,22 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Synapse.Common;
+
 namespace Microsoft.Health.Fhir.Synapse.DataClient.Models.DicomApiOption
 {
-    public class ChangeFeedOptions
+    public class BaseDicomApiOptions
     {
-        public ChangeFeedOptions(long offset, long limit, bool includeMetadata)
-        {
-            Offset = offset;
-            Limit = limit;
-            IncludeMetadata = includeMetadata;
-        }
+        public List<KeyValuePair<string, string>> QueryParameters { get; set; } = null;
 
-        public long Offset { get; set; }
+        public DicomApiVersion DicomVersion { get; set; } = DicomApiVersion.V1;
```

Comment:
Can we name the variable `DicomApiVersion` instead of `DicomVersion`? `DicomVersion` looks more like data version, such as R4, R5, while the value here actually refers to the API version of our DICOM Server.

Reply:
Updated, thanks.